### DISCUSSION TITLE
Resize handlers wrong position in absolute mode with spliter

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#4649] [Q&A] [answered]
Hi guys, I'm building a nice project using grapesjs and loving it, but I have an issue. Now I know that it was not intended to be used this way, but I abused it a bit and appended the toolbox into a splitter (left - tollbox, right - artboard). When the width of the artboard is changed, the 'gjs-tools' div position is calculated differently and it moves from the component.
I've attached a small screenshot.

<img width="846" alt="Screenshot 2022-10-07 at 15 09 02" src="https://user-images.githubusercontent.com/82867789/194550206-9ff3ae5d-9c2f-441c-8aa7-eb99af916bb2.png">

**Comment by artf** [comment, marked as answer]
You can use `editor.refresh()` when you apply changes that affect the canvas size.

**Reply by badiuciprian** [reply]
Thanks a lot, I will try it out
